app/templates/events/view.hbs

```diff
@@ -20,14 +20,22 @@
       <div class="twelve wide column {{unless device.isMobile 'right aligned'}}">
         {{#if device.isMobile}}
           <div class="ui icon fluid buttons">
-            <a href="{{href-to 'public' model.id}}" class="ui button">
+            <a href="{{href-to 'public' model.id}}" class="ui button" data-tooltip="Preview" data-position="top left">
               <i class="unhide icon"></i>
             </a>
-            <button class="ui button" {{action 'togglePublishState'}}>
-              <i class="{{if (eq model.state 'published') 'ban' 'check'}} icon"></i>
+              {{#if (eq model.state 'published')}}
+              <button class="ui button" {{action 'togglePublishState'}} data-tooltip="Unpublish" data-position="top left">
+                <i class="ban icon"></i>
             </button>
-            <button class="ui button"><i class="copy icon"></i></button>
-            <button class="ui red button" {{action 'openDeleteEventModal'}}><i class="trash icon"></i></button>
+              {{else}}
+              <button class="ui button" {{action 'togglePublishState'}} data-tooltip="Publish" data-position="top left">
+                <i class="check icon"></i>
+                </button>
+              {{/if}}
+            
+            <button class="ui button {{if isCopying 'loading'}}" {{action 'copyEvent'}} disabled="{{isCopying}}" data-tooltip="Copy" data-position="top left">
```

Comment:
Copy event isn't working in my local system. Please confirm.

Reply:
Wait will check

Reply:
@uds5501 That is the bug from server side , Events having Tickets in them are not getting copied and Tickets with External link are getting copied . Needs to be handle on different PR